daal4py/sklearn/_utils.py

```diff
@@ -95,7 +96,13 @@ def make2d(X):
 
 def get_patch_message(s):
     if s == "daal":
-        message = "uses Intel(R) oneAPI Data Analytics Library solver"
+        dev = None
+        if 'daal4py.oneapi' in sys.modules:
+            from daal4py.oneapi import _get_device_name_sycl_ctxt
+            dev = _get_device_name_sycl_ctxt()
+        if dev is None:
+            dev = "host"
+        message = f"uses Intel(R) oneAPI Data Analytics Library solver on {dev}"
```

Comment:
I would recommend not changing the message - when there is no context. Because this message can now be parsed by different users - including us.

Reply:
Corrected with latest recommendations